libcontainer/network_linux.go

```diff
@@ -52,15 +52,15 @@ func getNetworkInterfaceStats(interfaceName string) (*NetworkInterface, error) {
 	}
 	// Ingress for host veth is from the container. Hence tx_bytes stat on the host veth is actually number of bytes received by the container.
```

Comment:
Did you notice the comment here?  Do you think it is still valid?

Reply:
oops. I didn't see this comment 😂 

I think we use wrong function to collect network stat in vm-like container...

sorry for that.